style.css

```diff
@@ -0,0 +1,1628 @@
+  display: grid;
+  grid-template-rows: repeat(9, min-content);
+  width: 100vw;
+  font-weight: 300;
+  line-height: 1.6;
+}
+
+/* -------------------------NAVBAR--------------------- */
+
+.navigation__checkbox {
+  display: none;
+}
+
+.navigation__button {
+  background-color: #fff;
+  height: 5rem;
+  width: 5rem;
+  position: fixed;
+  top: 0.6rem;
+  right: 2rem;
+  z-index: 2000;
+  text-align: center;
+  cursor: pointer;
+}
+
+.navigation__background {
+  height: 4.5rem;
+  width: 4.5rem;
+  border-radius: 50%;
+  position: fixed;
+  top: 0.7rem;
+  right: 2.1rem;
+  background-image: linear-gradient(#f03e54, #ec2b4c, #d30a87, #cf0491);
+  transition: transform 0.8s cubic-bezier(0.86, 0, 0.07, 1);
+}
+
+.navigation__nav {
+  height: 100vh;
+  position: fixed;
+  top: 0;
+  left: 0;
+  z-index: 1500;
+  opacity: 0;
+  width: 0;
+  transition: all 0.8s cubic-bezier(0.68, -0.55, 0.265, 1.55);
+}
+
+.icons-navbar {
+  transform: translate(1rem, 2rem);
+}
+
+.icons-navbar-li {
+  background-color: #f03e54 !important;
+  margin: 1.5rem !important;
+}
+
+.svg-navbar {
+  width: 2rem !important;
+  height: 2rem !important;
+}
+
+.navigation__list-left {
+  position: absolute;
+  top: 50%;
+  left: 20%;
+  transform: translate(-20%, -50%);
+  list-style: none;
+  text-align: start;
+  width: 100%;
+}
+
+.navigation__list-right {
+  position: absolute;
+  top: 59%;
+  left: 70%;
+  transform: translate(-20%, -50%);
+  list-style: none;
+  text-align: start;
+  width: 100%;
+}
+
+.navigation__link-left:link,
+.navigation__link-left:visited {
+  display: inline-block;
+  font-size: 1.5rem;
+  font-weight: 300;
+  padding-left: 2rem;
+  padding-bottom: 1rem;
+  color: #fff;
+  text-decoration: none;
+}
+
+.navigation__link-right:link,
+.navigation__link-right:visited {
+  display: inline-block;
+  font-size: 1.5rem;
+  font-weight: 300;
+  padding-left: 2rem;
+  padding-bottom: 0.5rem;
+  color: #fff;
+  text-decoration: none;
+}
+
+.navigation__checkbox:checked ~ .navigation__background {
+  transform: scale(80);
+}
+
+.navigation__checkbox:checked ~ .navigation__nav {
+  opacity: 1;
+  width: 100%;
+}
+
+.navigation__icon {
+  position: relative;
+  margin-top: 2.5rem;
+}
+
+.navigation__icon,
+.navigation__icon::before,
+.navigation__icon::after {
+  width: 3rem;
+  height: 2px;
+  background-color: #333;
+  display: inline-block;
+  transition: all 0.2s;
+  z-index: 9999;
+}
+
+.navigation__icon::before,
+.navigation__icon::after {
+  content: "";
+  position: absolute;
+  left: 0;
+}
+
+.navigation__icon::before {
+  top: -0.8rem;
+}
+
+.navigation__icon::after {
+  top: 0.8rem;
+}
+
+.navigation__checkbox:checked + .navigation__button .navigation__icon {
+  background-color: transparent;
+}
+
+.navigation__checkbox:checked + .navigation__button .navigation__icon::before {
+  top: 0;
+  transform: rotate(135deg);
+}
+
+.navigation__checkbox:checked + .navigation__button .navigation__icon::after {
+  top: 0;
+  transform: rotate(-135deg);
```

Comment:
## Using Inefficient CSS Selectors[1]

When you are using selectors, the longer it is, the higher specificity it will comprise. This can break the CSS cascade and mess up the styles of your site.

This can be avoided by not using IDs within the selectors as they are directly specific.

- [ ]  Please keep `.navigation__icon` class and delete the left unnecessary rules

<hr>

_[1] [Good and bad  CSS practices](https://speckyboy.com/good-bad-css-practices/)_

Reply:
Thanks for the feedback! happy coding!